### ISSUE TITLE
Templates functionality

### ISSUE THREAD

**Issue by andrzejkupczyk** [andrzejkupczyk/mantisbt-todolists#14]
Allow to define default lists per issue type and append it automatically to a newly created issue.

**Comment by lgnutek** [comment]
This function would be perfect option. 

We're using MantisBT in our company mostly for managing process flow and we usually have the same "to-do-list" in every issue. With possibility to define default to-do list attached to every reported issue it would make this small but extremely useful plug-in for us.

Any chance for udpate in this matter?

**Comment by andrzejkupczyk** [comment]
Definitely. The further development of this plugin is strictly dependent on people's interest in using it. If this feature is desired, it is definitely worth adding 👍🏻
